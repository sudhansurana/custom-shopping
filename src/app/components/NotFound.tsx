import React from 'react'
import { Link } from 'react-router-dom'

export default function NotFound () {
  return (
    <div className="starter-template text-center py-5 px-3">
      <header>
        <h2>Page Not Found </h2>
      </header>
      <footer>
        <Link to='/'>Back to Home</Link>
      </footer>
    </div>
  )
}
