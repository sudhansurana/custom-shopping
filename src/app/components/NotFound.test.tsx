import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';
import {
  BrowserRouter as Router,
  Route,
  Switch,
  Link
} from 'react-router-dom'

test('NotFound component', () => {
  const component = renderer.create(
    <Router>
      <Switch>
          <Route path='**' component={NotFound}></Route>
        </Switch>
    </Router>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
