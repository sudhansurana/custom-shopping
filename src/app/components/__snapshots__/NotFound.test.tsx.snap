// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component 1`] = `
<div
  className="starter-template text-center py-5 px-3"
>
  <header>
    <h2>
      Page Not Found 
    </h2>
  </header>
  <footer>
    <a
      href="/"
      onClick={[Function]}
    >
      Back to Home
    </a>
  </footer>
</div>
`;
